Server/scripts/fetchYearMaps.ts

```diff
@@ -41,7 +41,7 @@ function timeFormatter (ms: number): string {
 
 const getModeDivison = memoizee(async (modeDivisionId: number) => {
     modeDivisionId += 1;
-    let mode = await ModeDivision.findOne(modeDivisionId + 1);
+    let mode = await ModeDivision.findOne(modeDivisionId);
```

Comment:
uhh, isn't the whole DB broken?

Reply:
no AFAIK i think this +1 was added at some point after the initial migration for some reason